### DISCUSSION TITLE
Twinkle Tray stops working after a while

### DISCUSSION THREAD

**Discussion by rschuech** [xanderfrangos/twinkle-tray#407] [Monitor Compatibility Issues]
I have three monitors connected to a Windows 11 desktop system:  two relatively old Asus VW266H displays and a newer Acer EB321HQU.   I don't seem to ever have any issues using Twinkle Tray with the two Asus screens.  However, while it works with the Acer for a while after the computer is started, I suspect that things break after the computer wakes from Sleep, though it might even be after Windows turns the displays turn off, before the computer even suspends to Sleep.  The Asus displays continue to respond to Twinkle Tray while the Acer does not.  Turning the Acer off and on via its power button has no effect, but I just realized that Refresh Displays by right clicking the app icon DOES fix the Acer.  

So, any ideas?  Twinkle Tray reports all three as using DDC/CI with a check mark even when the Acer stops responding to it.  In case it is relevant, the Acer is used as the primary monitor in Windows.  Ideally, a fix would be great, but a decent workaround would be some keyboard shortcut for Refresh Displays.  Thanks for any help and for a very nice app.

**Comment by xanderfrangos** [comment]
Hi @rschuech, 

Try turning on the "Get Current Brightness" option in the "DDC/CI Features" tab. Let me know if that helps or has no effect. Also, if you're able to determine if it happens because of sleep mode or the displays turning off, that would help. I can't actually detect when the screens turn off, but I can (and do) have some code to handle waking up from sleep. It's not perfect though.

**Comment by rschuech** [comment]
Thanks for the reply!  I turned on Get Current Brightness, no effect it seems.  I just did a simple test and set the screen to turn off in one minute in Windows power settings and this actually resulted in the problem, so it is just the display turning off it looks like, even before the system goes to sleep.

I did forget to mention that I am using a variety of cables - HDMI for one Asus, DVI for the other, and DisplayPort for the problematic Acer.  Would it be worth trying another type of cable or that shouldn't matter?

I wonder why only the Acer has this issue after turning off but the Asus don't.

**Reply by siddhantbapna** [reply]
Hey, have you found a fix for the issue?
Because I am also facing the same issue with my Acer issue.

**Reply by rschuech** [reply]
I'm using the current app version and I haven't noticed the problem lately.  Haven't changed anything except app updates and Windows updates. 🤷‍♂️

**Reply by JonyPlo** [reply]
I installed the app yesterday and I have the same problem on my samsung secondary monitor when my pc goes into sleep mode, the only solution is to restart my pc 😔
